pyproject.toml

```diff
@@ -11,8 +11,6 @@ classifiers = [
 
     "Intended Audience :: Developers",
 
-    "License :: OSI Approved :: MIT License",
```

Comment:
License doesn't change! I just got an error when building, as apparently license is now only under the `license` key (which we already had, and hasn't changed).